packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -4433,7 +4433,7 @@ describe('TableView', function () {
           let tree = render(
             <TableView aria-label="Table">
               <TableHeader>
```

Comment:
These two tests we added we actually identified as tests that would fail in the current table width layout code. They will work correctly with our new resizing code. 

Therefore for now we'll just add the "allowsResizing" prop to make sure they go through the correct path, but that prop could be removed in the future if you wanted to when the feature flag prop no longer exists.